package.json

```diff
@@ -8,6 +8,7 @@
   },
```

Comment:
Not in this PR, but I should really go through here and make sure everything is kosher. For example, there's probably some packages that aren't being used and some deps that would be in dev-deps